packages/custom-functions-metadata/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "custom-functions-metadata",
-  "version": "1.6.2",
```

Comment:
do not update the version, that will be done as part of the publish process.

Reply:
reverted